### PR TITLE
Lokalise latest june 4

### PR DESCRIPTION
**Pull request by alpita-masurkar**
#### Description:
The latest pull from lokalise.

### CODE REVIEW

File: android/app/src/bte/res/values/strings.xml [hunk 1]
```diff
@@ -3,7 +3,7 @@
   <!-- Shown on app info, app switcher, etc -->
   <string name="app_name" translatable="false">PathCheck BT</string>
   <!-- Shown below icon on launcher only -->
-  <string name="app_name_short" translatable="false">PathCheck BT</string>
+  <string name="app_name_short" translatable="false">Path Check BT</string>
```
Comment (JacobJaffe) [inline android/app/src/bte/res/values/strings.xml:6]: I believe we're sticking with `PathCheck` as official branding
Can we revert this file?

File: android/app/src/gps/res/values/strings.xml [hunk 2]
```diff
@@ -1,9 +1,9 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <resources>
   <!-- Shown on app info, app switcher, etc -->
-  <string name="app_name" translatable="false">PathCheck GPS</string>
+  <string name="app_name" translatable="false">PathCheck BT</string>
   <!-- Shown below icon on launcher only -->
-  <string name="app_name_short" translatable="false">PathCheck GPS</string>
+  <string name="app_name_short" translatable="false">Path Check BT</string>
```
Comment (JacobJaffe) [inline android/app/src/gps/res/values/strings.xml:6]: Looks like this file just got overridden to an old version. Can we revert this?

**Review by JacobJaffe — CHANGES_REQUESTED** [review]

File: android/app/src/bte/res/values/strings.xml (hunk 1 above)
Comment (alpita-masurkar) [reply android/app/src/bte/res/values/strings.xml:6]: yeah, you're right. this is an old PR though (just unmerged), we fixed it in the newer PRs. This should be reflected in the later PR.

File: android/app/src/gps/res/values/strings.xml (hunk 2 above)
Comment (alpita-masurkar) [reply android/app/src/gps/res/values/strings.xml:6]: hmm... this should have gotten pulled into Lokalise. will look into this.

**Comment by tstirrat** [comment]
superseded by #968

**Pull request closed without merging**